rostime/include/ros/duration.h

```diff
@@ -64,6 +64,16 @@ namespace ros
 ROSTIME_DECL void normalizeSecNSecSigned(int64_t& sec, int64_t& nsec);
 ROSTIME_DECL void normalizeSecNSecSigned(int32_t& sec, int32_t& nsec);
 
+// Arithmetic oprations of unsigned 32/64 bit numbers with these signed constants will convert these constants to
+// unsigned numbers according to the C standard. 16 bit unsigned numbers will get implicitly converted to int32_t for
+// the computation.
+constexpr int32_t USEC_TO_NSEC = 1000l;  //!< Number of nanoseconds in a microsecond.
```

Comment:
> use generic existing methods for converting time more generally.

Can you provide an example?

Reply:
Or do you suggest we should instead create an API for convenient conversions between `ros::Time` and `std::chrono`?

Reply:
Indeed I'd rather focus on creating a good bridge to std::chrono rather than trying to reimplement all the carefully thought out API that they've already implemented and built into the standard. It's a very large API that would turn into a large rabbit hole for us to fall down trying to replicate.